Integrations/integration-opswat-metadefender.yml

```diff
@@ -0,0 +1,239 @@
+  uses 30+ anti-malware engines to scan files for threats, significantly increasing
+  malware detection.
+configuration:
+- display: Server URL (e.g. https://api.metadefender.com/v2)
+  name: url
+  defaultvalue: https://api.metadefender.com/v2
+  type: 0
+  required: true
+- display: API Key
+  name: api
+  defaultvalue: ""
+  type: 4
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |
+    var sendRequest = function(url,param, fileId) {
+        var requestUrl = params.url + url + param;
+        var res;
+
+        if (fileId) {
+            var fileName = dq(invContext, 'File(val.EntryID == \'' + fileId + '\').Name');
+            res = httpMultipart(
+                requestUrl,
+                fileId,
+                {
+                    Method: 'POST',
+                    Headers: {
+                        Accept: ['application/json'],
+                        filename: fileName
+                    }
+                },
+                {},
+                params.insecure,
+                params.proxy,
+                undefined,
+                'file',
+                fileName);
+        } else {
+           res = http(
+            requestUrl,
+            {
+                Method: 'GET',
+                Headers: {
+                    Accept: ['application/json']
+                }
+            }
+        );
+        }
+
+
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+
+        return res;
+    };
+
+
+    var searchHash = function() {
+        var res = JSON.parse(sendRequest('/hash/', args.hash).Body);
+        var md = '# OPSWAT-Metadefender\n';
+        var ec = {};
+        var dbotScore = 0;
+
+        md += '### Results for Hash ' + args.hash + '\n';
+
+        if (res.file_info) {
+            md += 'File name: ' + res.file_info.display_name + '\n';
+            md += 'File description: ' + res.file_info.file_type_description + '\n';
+            md += 'Scan result: ' + res.scan_results.scan_all_result_a + '\n';
+            md += 'Detected AV: ' + res.scan_results.total_detected_avs + '/' + res.scan_results.total_avs + '\n';
+
+            md += 'AV Name|Def Time|Threat Name Found\n';
+            md += '---|---|---\n';
+
+            var avRes = res.scan_results.scan_details;
+
+            Object.keys(avRes).forEach(function(key) {
+              md += key + '|';
+              md += avRes[key].def_time + '|';
+              md += avRes[key].threat_found + '\n';
+            });
+
+            var percntBad = (res.scan_results.total_detected_avs / res.scan_results.total_avs) * 100;
+
+            if (percntBad >= 66) {
```

Comment:
66 and 34 should defined as arguments and not hardcoded.

Reply:
Added